style.css

```diff
@@ -68,9 +67,9 @@ body{
 	text-decoration: none;
 	padding: 1rem;
 	display: block;
-	color: black;
-	font-family: 'Lexend', sans-serif;
-	font-weight: 400;
+	color: white;
```

Comment:
#changed most of my text to white and blue instead of black to suit my dark theme